source/class/qx/theme/indigo/ColorDark.js

```diff
@@ -85,8 +85,8 @@ qx.Theme.define("qx.theme.indigo.ColorDark", {
     "text-placeholder": "#cbc8cd",
 
     // tooltip
-    "tooltip": "#dddddd",
-    "tooltip-text": "#000000",
+    "tooltip": "#000000",
```

Comment:
I think less harsh color than a pure black color for background would be nice some kind of gray. Tried #666666 - probably looks better than black.

Reply:
Updated to gray.